src/experimental/apply-style.ts

```diff
@@ -1,32 +1,66 @@
-import Yoga from 'yoga-layout-prebuilt';
-
-const applyPositionStyles = (node, style) => {
+import Yoga, {YogaNode} from 'yoga-layout-prebuilt';
+import {
+	PositionStyles,
+	MarginStyles,
+	PaddingStyles,
+	FlexStyles,
+	DimensionStyles,
+	Styles
+} from '../styles';
+
+const applyPositionStyles = (node: Yoga.YogaNode, style: PositionStyles) => {
 	if (!style.position) {
-		node.setPositionType(NaN);
+		node.setPositionType(Yoga.POSITION_TYPE_RELATIVE);
 	}
 
 	if (style.position === 'absolute') {
 		node.setPositionType(Yoga.POSITION_TYPE_ABSOLUTE);
 	}
 };
 
-const applyMarginStyles = (node, style) => {
-	node.setMargin(Yoga.EDGE_START, style.marginLeft || style.marginX || style.margin || 0);
-	node.setMargin(Yoga.EDGE_END, style.marginRight || style.marginX || style.margin || 0);
-	node.setMargin(Yoga.EDGE_TOP, style.marginTop || style.marginY || style.margin || 0);
-	node.setMargin(Yoga.EDGE_BOTTOM, style.marginBottom || style.marginY || style.margin || 0);
+const applyMarginStyles = (node: Yoga.YogaNode, style: MarginStyles) => {
+	node.setMargin(
+		Yoga.EDGE_START,
+		(style.marginLeft ?? style.marginX) ?? style.margin ?? 0
```

Comment:
Could you keep the old syntax with `||` operator? I find `??` confusing when used multiple times at once.

Reply:
This was auto corrected by xo. I would need to add an ignore or change config. Would you like me to change it?

Reply:
Yes please, I find it more noisy than the original `||` version :)

Reply:
Done in https://github.com/vadimdemedes/ink/pull/264/commits/bd1c5100cbc45cdab3326334f2629b9744deb903